types/react-router-redux/index.d.ts

```diff
@@ -54,12 +54,12 @@ export interface LocationActionPayload {
     args?: any[];
 }
 
-export interface RouterAction extends Action {
+export interface RouterAction extends AnyAction {
```

Comment:
This removes type safety from objects of this interface:

```ts
declare const action: RouterAction;

action.foo.bar;  // no error
```

I don't think that we even need to extend `Action`, since loose `RouterAction` would still be assignable to `Action` and `AnyAction`.